egs/csj/s5/local/nnet3/run_tdnn_swbd.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
```

Comment:
Yes, run_tdnn.sh gives better results than the baseline. So we will remove the baseline.